### DISCUSSION TITLE
Can you link a Lecture to an LTI Tool Consumer?

### DISCUSSION THREAD

**Discussion by acbart** [ls1intum/Artemis#9018] [Q&A]
I see that it is possible to link a single exercise to moodle/canvas, but is it possible to link a Lecture that contains multiple exercises? Or, more generally, is there a way to link a sequence of exercises to a single canvas assignment via LTI?

**Comment by MaximilianAnzinger** [comment]
Hello @acbart ,

We're currently working on extending our LTI capabilities.
In the near future you will be able to link:
- exercises
- lectures
- lecture units
- competencies
- learning paths
- learning analytics dashboard

Additionally, we're planning to integrate Iris (our intelligent tutoring system) such that users that are not natively working on Artemis are still able to benefit from all it's capabilities.
